data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,207 @@ mission "Arachi Cartoonist 2"
+
+mission "Coalition: Council of Ahr: Drama"
+	invisible
+	landing
+	to offer
+		or
+			has "Coalition: Council of Ahr: Bebliss: active"
+			has "Coalition: Council of Ahr: Chamgar: active"
+		has "Coalition: Council of Ahr: active"
+	source
+		planet "Hammer of Debrugt" "Stronghold of Flugbu"
+	destination "Ahr"
+	on enter
+		conversation
+			`After departing from <origin>, you start hearing some shouting from outside your cockpit, so you leave to check up on your passengers. Plubat, from House Chamgar, is banging one leg against the door to the room of Plubat, from House Bebliss. "Captain! Good thing that here, you are," he says. "Sent an envoy to mock me, that damned house Bebliss has! Copying my name, she is!"`
+			`	Plubat, from House Bebliss, opens the door to her room to respond. "Nonsense that is, Captain. Look at us, several decades older, I am, so copying you, how could I be? The offending party here, house Chamgar is."`
+			`	"Changed your name for this mockery, you could have. Know of your house's grudge against us, I do," he replies.`
+			`	"Absurd, childish, that is. The one to start mocking me, you were," she says. "Captain <last>, urge you to make haste for Ahr, I must. Put up with this buffoon for much longer, I cannot."`
+			`	Lady Plubat closes the door to her room again, and you convince house Chamgar's representative to also go back to his room. You return to your cockpit, hoping that's the last of that.`
+	on enter "Sol Arach"
+		dialog
+			`In spite of your best attempts to keep them away from each other, Plubat and Plubat have been brief on their breaks from insulting one another. However, they head back to their rooms on their own once you enter this system, without you needing to leave the cockpit to interfere. Being so close to their destination, you hope they won't give you any more headaches.`
+
+mission "Coalition: Council of Ahr: On Complete"
+	invisible
+	landing
+	to offer
+		has "Coalition: Council of Ahr: done"
+	source "Ahr"
+	on offer
+		conversation
+			`A few minutes after the final representative leaves your ship, you receive your payment of 4,500,000 credits. It comes with a message from the royal family, with the symbols of all Arach houses on it. You recognize most from advertisements or offices on some planets, but it's your first time seeing the center one. You imagine it's the emblem of the royal family.`
+			scene "scene/councilofahr2"
+			`	"It is with great honor that we salute you, Captain <last>, as a friend and assistant to the Council of Ahr. Although certain that, to you, such kinds of jobs can appear mundane, it can be difficult, at times, to arrange for appropriate transportation for our compatriots from all great houses, so know that house Oberegt is deeply grateful for your cooperation."`
```

Comment:
```suggestion
			`	"It is with great honor that we salute you, Captain <last>, as a friend and assistant to the Council of Ahr. Although certain that, to you, such kinds of jobs can appear mundane, it can be difficult, at times, to arrange for appropriate transportation for our compatriots from all great houses, so know that House Oberegt is deeply grateful for your cooperation."`
```